### PR TITLE
Update 'How to Preview' section for Macs

### LINKED ISSUE #7: Serving static sites on Mac OS with Python
For this section (https://github.com/exalted/mdwiki-seed#mac-os), I would also suggest using python to serve.

You just have to navigate to the folder in the terminal and type `python -m SimpleHTTPServer 8000`.

All newer Macs come with python. More about it here: https://docs.python.org/2/library/simplehttpserver.html

I've created a pull request for this here: https://github.com/exalted/mdwiki-seed/pull/8

### PR DESCRIPTION
**Pull request by bplmp**
Suggest using python simpleserver to serve static site on Mac.
## 

Close #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,8 @@ Mac OS
 
 The easiest way to serve up static sites on a Mac is to use [Anvil](http://anvilformac.com). Go ahead and download it from their website, install and add a site using the status bar icon: simply select the folder where your wiki is located on your Mac.
 
+If you don't want to donwload any apps, you can use the Terminal on your Mac and pythong. To do this, go to your site folder in the Terminal app and type `python -m SimpleHTTPServer 8000`. This will start running a local server which you can navigate to by typing the URL localhost:8000 in any web browser.
```
Comment (exalted) [inline README.md]: If you could please fix minor typos:
- donwload `->` download
- pythong `->` Python (note capital P)

Also please consider backtick-wrapping "localhost:8000" as in `localhost:8000`.

**Comment by exalted** [comment]
:clap: for this PR, I really appreciate what you did here. I just commented small things I noticed [here](https://github.com/exalted/mdwiki-seed/pull/8/files#r49089176). If you could make those changes as well, I will gladly merge this ASAP.

:beers:

**Comment by bplmp** [comment]
Sure, just made the suggested changes here https://github.com/bplmp/mdwiki-seed/commit/a3d24b89b0175c740aa2fbef5cb682bbfca54253.

Happy to contribute!

**Comment by exalted** [comment]
:sparkles: work, thank you!

**Comment by bplmp** [comment]
:beers: !

**Pull request merged**